testslide/mock_callable.py

```diff
@@ -713,6 +713,7 @@ def to_return_value(self, value):
         """
         Always return given value.
         """
+        _validate_return_type(self._original_callable, value)
```

Comment:
This is not enough: other stuff such as `.to_return_values()` won't work. I believe that if we add it to `_CallableMock.__call__`, it should cover 100% of cases.

I'll cut a PR on top, adding tests specifically for the changes here, covering all cases.